CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/entity/UserEntity.kt

```diff
@@ -0,0 +1,24 @@
+package co.nimblehq.coroutine.entity
+
+data class UserEntity(
+    val id: Int?,
+    val name: String,
+    val username: String,
+    val email: String?,
+    val address: Address?,
+    val phone: String,
+    val website: String
+) {
+    data class Address(
```

Comment:
No biggie, but our [convention](https://nimblehq.co/compass/development/code-conventions/kotlin/#formatting) requires a blank line after the class declaration and before the first line of code inside the class. 😄